spec/spec_helper.rb

```diff
@@ -398,14 +265,34 @@ def rpm_install_tests(options = {})
   }
 end
 
+def deb_install_tests(options = {})
+  # Extract params
+  deb_repo = options.fetch(:deb_repo, 'ppa:odl-team/boron')
+
+  # Confirm the presence of Deb-related resources
+  it { should contain_apt__ppa(deb_repo) }
+  it { should contain_package('opendaylight') }
+
+  # Confirm relationships between Deb-related resources
+  it { should contain_package('opendaylight').that_requires("Apt::Ppa[#{deb_repo}]") }
+  it { should contain_apt__ppa(deb_repo).that_comes_before('Package[opendaylight]') }
+
+  # Confirm presence of Deb-related resources
+  it {
+    should contain_package('opendaylight').with(
+      'ensure'   => 'present',
+    )
+  }
+end
+
 # Shared tests for unsupported OSs
 def unsupported_os_tests(options = {})
   # Extract params
   expected_msg = options.fetch(:expected_msg)
   rpm_repo = options.fetch(:rpm_repo, 'opendaylight-5-testing')
 
   # Confirm that classes fail on unsupported OSs
-  it { expect { should contain_class('opendaylight') }.to raise_error(Puppet::Error, /#{expected_msg}/) }
+  it { expect { should contain_class('opendaylight') }.to raisehould work idempotently with no errors_error(Puppet::Error, /#{expected_msg}/) }
```

Comment:
Ah, I think this is just a typo, copied in "hould work idempotently with no errors" accidently